<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimationDialog</class>
 <widget class="QDialog" name="AnimationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>wrlAnimation</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qWrlAnimation.qrc">
    <normaloff>:/CC/plugin/qWrlAnimation/images/icon.png</normaloff>:/CC/plugin/qWrlAnimation/images/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Camera name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Output directory</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Input directory </string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QToolButton" name="outputDirButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QToolButton" name="inputDirButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="inputDirLineEdit"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="outputDirLineEdit"/>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="cameraNameLineEdit">
       <property name="text">
        <string>Camera001</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Relative position</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>X</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="xDoubleSpinBox">
           <property name="minimumSize">
            <size>
             <width>115</width>
             <height>25</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>115</width>
             <height>25</height>
            </size>
           </property>
           <property name="decimals">
            <number>5</number>
           </property>
           <property name="minimum">
            <double>-9999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>9999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Z</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="yDoubleSpinBox">
           <property name="minimumSize">
            <size>
             <width>115</width>
             <height>25</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>115</width>
             <height>25</height>
            </size>
           </property>
           <property name="decimals">
            <number>5</number>
           </property>
           <property name="minimum">
            <double>-9999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>9999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Y</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="zDoubleSpinBox">
           <property name="minimumSize">
            <size>
             <width>115</width>
             <height>25</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>115</width>
             <height>25</height>
            </size>
           </property>
           <property name="decimals">
            <number>5</number>
           </property>
           <property name="minimum">
            <double>-9999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>9999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Relative rotation</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>X</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="xRotDoubleSpinBox">
           <property name="minimumSize">
            <size>
             <width>115</width>
             <height>25</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>115</width>
             <height>25</height>
            </size>
           </property>
           <property name="decimals">
            <number>5</number>
           </property>
           <property name="minimum">
            <double>-180.000000000000000</double>
           </property>
           <property name="maximum">
            <double>180.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Z</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="yRotDoubleSpinBox">
           <property name="minimumSize">
            <size>
             <width>115</width>
             <height>25</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>115</width>
             <height>25</height>
            </size>
           </property>
           <property name="decimals">
            <number>5</number>
           </property>
           <property name="minimum">
            <double>-180.000000000000000</double>
           </property>
           <property name="maximum">
            <double>180.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Y</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="zRotDoubleSpinBox">
           <property name="minimumSize">
            <size>
             <width>115</width>
             <height>25</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>115</width>
             <height>25</height>
            </size>
           </property>
           <property name="decimals">
            <number>5</number>
           </property>
           <property name="minimum">
            <double>-180.000000000000000</double>
           </property>
           <property name="maximum">
            <double>180.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Output resolution</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sizeModeComboBox">
       <property name="toolTip">
        <string>List of static resolution</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <item>
        <property name="text">
         <string>1920x1080</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3840x2160</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5760x3240</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7680x4320</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9600x5400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11520x6480</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>13440x7560</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15360x8640</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16320x9180</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16384x9216</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ratioCheckBox">
       <property name="text">
        <string>Ratio 1:1</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Fps</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="fpsSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Watermark file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="watermarkFileLineEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="watermarkButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="previewButton">
       <property name="toolTip">
        <string>Preview the animation</string>
       </property>
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportFramesPushButton">
       <property name="toolTip">
        <string>Export frames as individual images</string>
       </property>
       <property name="text">
        <string>Export frames</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qWrlAnimation.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AnimationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AnimationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
